admin/error-types/mtc-school-mismatch.error.js

```diff
@@ -0,0 +1,10 @@
+const mtcBaseError = require('./mtc-base.error')
```

Comment:
I would prefer to have all the error classes, including the base class defined in one file.  This decreases the maintenance burden.

```
const { MtcSpecificError } = require('mtc-error')
...
throw new MtcSpecificError(message)
```

Reply:
I don't mind that change but then we have a single directory which will contain only 1 file.

Reply:
@jon-shipley done